pkg/machines/components/vmOverviewTab.jsx

```diff
@@ -23,7 +23,7 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
-const Items = ({ items, idPrefix, divider, colClass }) => {
+const Items = ({ items, idPrefix }) => {
```

Comment:
not important, but the commit message says "These very properties" but it means "were"

Reply:
Fixed